config/robotframework_aio/release_items_RobotFramework_TestsuitesManagement.json

```diff
@@ -46,14 +46,12 @@
 
    "RELEASES"  : {
                    "0.9." : [
-                              "RobotFramework_TestsuitesManagement change 0.9 A",
-                              "RobotFramework_TestsuitesManagement change 0.9 B",
-                              "RobotFramework_TestsuitesManagement change 0.9 X"
+                              "RobotFramework_TestsuitesManagement change 0.9 New RobotFramework 6.1 adaptation; Maintenance of log output"
                             ],
                    "0.10." : [
-                               "RobotFramework_TestsuitesManagement change 0.10 A",
-                               "RobotFramework_TestsuitesManagement change 0.10 B",
-                               "RobotFramework_TestsuitesManagement change 0.10 X"
+                               "RobotFramework_TestsuitesManagement change 0.10 Improve dotdict feature",
+                               "RobotFramework_TestsuitesManagement change 0.10 Update logging",
+                               "RobotFramework_TestsuitesManagement change 0.10 Code maintenance"
                              ]
```

Comment:
Hi Son,

"improve", "update", "maintenance". OK, a lot of things happened. And should be mentioned. But what exactly is the outcome for users? These terms are so much common. No one will get a concrete idea about the changes. Is there any chance to go a little bit more in detail? What is improved? What exactly has been changed in the logging?